tinygrad/runtime/ops_gpu.py

```diff
@@ -9,26 +9,24 @@
 from tinygrad.codegen.cstyle import CStyleCodegen, CStyleLanguage
 
 OSX_TIMING_RATIO = (125/3) if OSX else 1.0   # see test/external_osx_profiling.py to determine this ratio. it's in like GPU clocks or something
-FLOAT16 = getenv("FLOAT16", 0)
 
 # TODO: if you fork and exit the child process after creating anything with cl on AMD, it hangs on e.wait()
 ROCM_LLVM_PATH = pathlib.Path("/opt/rocm/llvm/bin")
 #ROCM_LLVM_PATH = pathlib.Path(__file__).parent.parent.parent.parent / "extra/rocm/build/llvm-project/bin"
 if DEBUG >= 5:
-  from extra.helpers import enable_early_exec
-  early_exec = enable_early_exec()
+  early_exec = fromimport("extra.helpers", "enable_early_exec")()
 
 class _CL:
   def __init__(self):
-    platforms: List[List[cl.Device]] = [y for y in ([x.get_devices(device_type=cl.device_type.GPU) for x in cl.get_platforms()] + [x.get_devices(device_type=cl.device_type.CPU) for x in cl.get_platforms()]) if len(y)]
-    if DEBUG >= 1: print(f"using {platforms[getenv('CL_PLATFORM', 0)]}")
-    self.cl_ctx: cl.Context = cl.Context(devices=[x for x in platforms[getenv('CL_PLATFORM', 0)] if x.name not in getenv('CL_EXCLUDE', "").split(",")])
+    platform: List[cl.Device] = [y for y in ([x.get_devices(device_type=cl.device_type.GPU | cl.device_type.CPU) for x in cl.get_platforms()]) if len(y)][getenv('CL_PLATFORM', 0)]
```

Comment:
This changes the behaviour of this line. Originally they are split to put CPU devices on a lower priority than GPU devices.

Reply:
Ahh, makes sense.  I'll revert to the original logic here.